src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -186,6 +189,14 @@ def put(self, request, organization, version):
                         datetime=release.date_released,
                     )
 
+            analytics.record(
+                "release.set_commits",
+                user_id=request.user.id if request.user and request.user.id else None,
+                organization_id=organization.id,
+                project_ids=[project.id for project in release.projects.all()],
```

Comment:
@NisanthanNanthakumar I don't love that we are running additional queries just to get the projects. I also don't know how well having an array of projects would work. Maybe we could just remove it.

Reply:
@scefali I've refactored to reuse the results from the same query made elsewhere